pkg/apis/garden/v1beta1/types.go

```diff
@@ -994,6 +1001,11 @@ type OpenStackWorker struct {
 type Local struct {
 	// Networks holds information about the Kubernetes and infrastructure networks.
 	Networks LocalNetworks `json:"networks"`
+	// MachineImage holds information about the machine image to use for all workers.
+	// It will default to the first image stated in the referenced CloudProfile if no
```

Comment:
Same.

Reply:
Not sure what is expected here. Can you please elaborate?